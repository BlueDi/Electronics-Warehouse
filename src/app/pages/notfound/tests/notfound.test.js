import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';
import NotFound from '@pages/notfound';

test('<NotFound />', () => {
  const component = renderer.create(
    <Router>
      <NotFound />
    </Router>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
