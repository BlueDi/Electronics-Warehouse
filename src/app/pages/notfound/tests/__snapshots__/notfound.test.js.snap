// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> 1`] = `
<h3>
  404 - Page Not Found.
</h3>
`;
